chrome/content/zotero/xpcom/data/item.js

```diff
@@ -1396,9 +1396,11 @@ Zotero.Item.prototype._saveData = Zotero.Promise.coroutine(function* (env) {
 			
 			let newParentItemNotifierData = {};
 			//newParentItemNotifierData[newParentItem.id] = {};
-			Zotero.Notifier.queue(
-				'modify', 'item', parentItemID, newParentItemNotifierData, env.options.notifierQueue
-			);
+			if (!env.options.skipNotifier) {
+				Zotero.Notifier.queue(
+						'modify', 'item', parentItemID, newParentItemNotifierData, env.options.notifierQueue
+						);
```

Comment:
Here and elsewhere, keep original style (one tab, closing paren aligned with opening line)